package.json

```diff
@@ -12,7 +12,8 @@
     },
     "./examples/fonts/*": "./examples/fonts/*",
     "./examples/jsm/*": "./examples/jsm/*",
-    "./src/*": "./src/*"
+    "./src/*": "./src/*",
+    "./package.json": "./package.json" 
```

Comment:
I'm not sure we want to support this. The argumentation in https://github.com/nodejs/node/issues/33460#issuecomment-630506507 makes totally sense to me.

> The goal of exports is to fully encapsulate the public API of a package in a way that allows sound analysis of execution, optimization, breaks etc etc. Exposing the package.json goes against this by making the properties of the package.json part of the public API.

I think this issue should be addressed by the bundler/built tool. It should access the `package.json` not over the `exports` section.